breadcord/core_modules/module_manager/__init__.py

```diff
@@ -86,7 +87,7 @@ async def install(self, interaction: discord.Interaction, module: str, branch: s
             f'https://api.github.com/repos/{module}/contents/manifest.toml' + (f'?ref={branch}' if branch else ''),
             headers={'Accept': 'application/vnd.github.raw'},
         ) as response:
-            if response.status != 200:
+            if response.status != HTTPStatus.OK:
```

Comment:
Silly question, but is this valid? Does aiohttp do some helpful handling there, or would we have to use `HTTPStatus.OK.value` instead? It makes logical sense but I'm not sure about how thorough you were with testing, so just to be safe

Reply:
```py
>>> import http
>>> 200 == http.HTTPStatus.OK
True
```